code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -290,14 +313,15 @@
 
	volume = 120
 
	flags = OPENCONTAINER
 

```

Comment:
Compare to [here](https://github.com/ParadiseSS13/Paradise/pull/783), this was not here prior to this PR.